src/pages/Food/index.jsx

```diff
@@ -12,25 +13,34 @@ export default function Food() {
   const [dataLunch, setDataLunch] = useState([]);
   const [dataDesserts, setDataDesserts] = useState([]);
   const [dataDrinks, setDataDrinks] = useState([]);
+  const user = useSelector((state) => state.user);
 
   useEffect(() => {
-    axios.get('http://localhost:5000/food/lunch').then(({ data }) => {
+    api.get(`/food/lunch`).then(({ data }) => {
       setDataLunch(data);
+      console.log(user);
```

Comment:
??? :/

Reply:
i promise you i deleted before push wtf

Reply:
I re-pushed, it should be ok

Reply:
Alright, I think I was reviewing the pr when you pushed